model/import/AssemblerService.php

```diff
@@ -57,6 +72,21 @@ class AssemblerService extends ConfigurableService implements AssemblerServiceIn
 
     const OPTION_FILESYSTEM_ID = 'filesystemId';
 
+    /**
+     * Getting file content
+     */
+    const OPTION_FILE_READER = 'fileReader';
+
+    /**
+     * Transform runtime to the different formats
+     */
+    const OPTION_SERVICE_CALL_CONVERTER = 'serviceCallConverter';
```

Comment:
As we discussed it's not necessary to keep backward compatibility for already exported delivery assemblies with PHP serialized ServiceCall runtime. We can keep those methods as adapters for json encode/decode methods. 
With this change in requirements we don't need serviceCallConverter classes and configurable option.